contracts/contracts/PoolFeeManagerProxy.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
+import "./Interfaces.sol";
+
+/** 
+ * @title   PoolFeeManagerProxy
+ * @author  AuraFinance
+ * @notice  Wraps PoolManagerV4 and Booster Fee Manager capabilites 
+ *           It allows to add pools and set their reward multiplier.
+ */
+contract PoolFeeManagerProxy{
+
+    address public immutable poolManager;
+    address public immutable booster;
+    address public operator;
+
+    bool public protectAddPool;
+
+    /// @dev The value must be within the allowed range on Booster contract.
+    uint256 public defaultRewardMultiplier;
+    /// @dev Same value as Booster.REWARD_MULTIPLIER_DENOMINATOR
+    uint256 public constant REWARD_MULTIPLIER_DENOMINATOR = 10000;
+
+    event DefaultRewardMultiplierUpdated(uint256 newRewardMultiplier);    
+
+    /**
+     * @param _poolManager      Currently PoolManagerV4
+     * @param _booster          Booster address
+     * @param _operator         Convex multisig
+     */
+    constructor(
+        address _poolManager,
+        address _booster,
+        address _operator
+    ) public {
+        poolManager = _poolManager;
+        booster = _booster;
+        operator = _operator;
+        protectAddPool = true;
+    }
+
+    //-------------------------------------//
+    //---------- PoolManagerV4 ------------//
+    //-------------------------------------//
+    function setPoolManagerOperator(address _operator) external {
+        require(msg.sender == operator, "!auth");
+        IPoolManager(poolManager).setOperator(_operator);
+    }
+
+    function setPoolManagerProtectPool(bool _protectAddPool) external {
+        require(msg.sender == operator, "!auth");
+        IPoolManager(poolManager).setProtectPool(_protectAddPool);
```

Comment:
Shouldn't this always stay protected? Because the PoolFeeManagerProxy needs to be the only thing that can addPools? We can probably remove this function